keyboards/flygone60/rev3/readme.md

```diff
@@ -0,0 +1,19 @@
+# flygone60
+
+![Flygone60](https://i.imgur.com/x63gMG2h.jpeg)
+
+A 60% that takes arrow keys seriously!
+
+* Keyboard Maintainer: [ShandonCodes](https://github.com/ShandonCodes)
+* Hardware Supported: Flygone60
+* Hardware Availability: ![Kit](https://www.etsy.com/listing/1025415290/flygone60-keyboard-kit?ga_order=most_relevant&ga_search_type=all&ga_view_type=gallery&ga_search_query=flygone60&ref=sr_gallery-1-13&organic_search_click=1&frs=1)
```

Comment:
```suggestion
* Hardware Availability: [Kit](https://www.etsy.com/listing/1025415290/flygone60-keyboard-kit?ga_order=most_relevant&ga_search_type=all&ga_view_type=gallery&ga_search_query=flygone60&ref=sr_gallery-1-13&organic_search_click=1&frs=1)
```
This link does not work because of superfluous `!` sign (`!` is used for embedding an image and needs a direct link to image; a link to a web page does not need that extra markup).

Reply:
I see. Thanks for the catch!